tools/benchmark_tool/openvino/tools/__init__.py

```diff
@@ -1,3 +1,10 @@
 # Copyright (C) 2018-2023 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
-__path__ = __import__('pkgutil').extend_path(__path__, __name__)
+
+__path__ = __import__("pkgutil").extend_path(__path__, __name__)
+
+try:
+    import openvino.tools.mo
```

Comment:
what's about POT and other tools? should we try to import it as well? if no - why?

Reply:
Please check messages about the topic. I called for all requests, awaiting responses.

Reply:
I don't understand what I should check
I've asked 2 questions and have not received the answers